src/Hydraulic/IsothermalCompressible/components.jl

```diff
@@ -463,16 +464,14 @@ dm ────►               │  │ area
 
 See also [`FixedVolume`](@ref), [`DynamicVolume`](@ref)
 """
-@component function Volume(;
+@mtkmodel Volume begin
 
-        #parameters
-        area,
-        direction = +1, name)
-    pars = @parameters begin
-        area = area
+    @parameters begin
+        area
+        direction = +1
```

Comment:
`direction` could be of type `@structurtal_parameters`.

Is this intentionally promoted to a `@parameters`?

While here, could you change +1 to 1?